src/main/java/com/roommatematcher/enterprise/RoomMateController.java

```diff
@@ -1,16 +1,69 @@
 package com.roommatematcher.enterprise;
 
+import com.roommatematcher.enterprise.dto.Profile;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Controller;
-import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.*;
 
+/**
```

Comment:
Implemented Java doc over the RoomateController class to outline its basic functionality.

Reply:
Avoid using the * wildcard import.  Specifically identifying what you are importing is better for documentation, and avoids clashes.  Example: java.util.* and java.sql.* but have a class named Date.  If you import both with wildcard, and then reference class Date, which one should it use?